code/modules/cm_marines/marines_consoles.dm

```diff
@@ -91,18 +91,11 @@
 					printing = TRUE
 					playsound(src.loc, 'sound/machines/fax.ogg', 15, 1)
 					sleep(40)
-					var/faction = "N/A"
-					if(target_id_card.faction_group && islist(target_id_card.faction_group))
-						faction = jointext(target_id_card.faction_group, ", ")
-					if(isnull(target_id_card.faction_group))
-						target_id_card.faction_group = list()
-					else
-						faction = target_id_card.faction_group
 					var/contents = {"<center><h4>Access Report</h4></center>
 								<u>Prepared By:</u> [user_id_card?.registered_name ? user_id_card.registered_name : "Unknown"]<br>
 								<u>For:</u> [target_id_card.registered_name ? target_id_card.registered_name : "Unregistered"]<br>
 								<hr>
-								<u>Faction:</u> [faction]<br>
+								<u>Faction:</u> [target_id_card.faction ? target_id_card.faction : "N/A"]<br>
```

Comment:
These changes do not look to be correct. See your own testing example image where it says the faction is a list.

Reply:
Changes are after the image, deleted stuff directly above this line fixes that for some reason it was being set to a list variable.

Reply:
Image updated